frontend/src/app/oe/components/strike-detail/strike-detail.component.ts

```diff
@@ -184,6 +207,16 @@ export class StrikeDetailComponent implements OnInit, OnDestroy {
         this.nextBlockHeight = fromBlock.height + 1;
         this.setNextAndPreviousBlockLink();
 
+        const strikesFilter = {
+          creationTimeEQ: fromBlock.timestamp,
```

Comment:
there is no `creationTimeEQ` filter field. You probably meant `strikeMediantimeEQ` field instead:
```
filter
string(${"personEQ":"b8ab3013e4adb35fae6cbdc9d84c86cd280157b7a93b984c0b40baf7f21b8f72", "personNEQ":"b8ab3013e4adb35fae6cbdc9d84c86cd280157b7a93b984c0b40baf7f21b8f72", "guessEQ":"slow", "guessNEQ":"fast", "observedResultEQ":"slow", "observedResultNEQ":"fast", "strikeBlockHeightGTE":1, "strikeBlockHeightLTE":1, "strikeBlockHeightEQ":1, "strikeBlockHeightNEQ":1, "strikeMediantimeGTE":1, "strikeMediantimeLTE":1, "strikeMediantimeEQ":1, "strikeMediantimeNEQ":1, "sort":"descend", "class":"guessable"}, )
```

Reply:
Thank s @dambaev it is resolved